executor/common_linux.h

```diff
@@ -1327,6 +1409,56 @@ static void netlink_wireguard_setup(void)
 	close(sock);
 }
 
+#if SYZ_EXECUTOR || SYZ_NIC_VF
+static void netlink_nicvf_setup(void)
```

Comment:
Here you might also need the `flag_nic_vf` check

Reply:
@a-nogikh Requested change made.

Reply:
It will likely fail to compile now if `SYZ_EXECUTOR` is not defined (e.g. for reproducers).

Reply:
@a-nogikh Should it just be:

Reply:
It should be `#if SYZ_EXECUTOR` since the `flag_nic_vf` is defined in `executor/executor.cc`

Reply:
You did it the right way in `find_vf_interface`

Reply:
@a-nogikh Made change as you suggested. See: https://github.com/google/syzkaller/commit/daf311629d933a5cfd78b10447fdb0c709a5e7f8